bin/main.rb

```diff
@@ -1,47 +1,58 @@
 #!/usr/bin/env ruby
-# rubocop:disable Style/Next
 
-puts 'Welcome to the game of Tic Tac Toe!!'
-puts 'Player 1 what is your name?'
-player1 = gets.chomp
-puts "Welcome #{player1} choose your game symbol: X or O"
-player_1_symbol = gets.chomp
-# player 1 chooses "X or O"
-puts "Ok #{player1} you will be playing with #{player_1_symbol} as your symbol"
-puts 'Player 2 what is your name?'
-player2 = gets.chomp
-puts "Welcome #{player2} choose your game symbol"
-player_2_symbol = gets.chomp
-# player 2 chooses "O or X"
-puts "Ok #{player2} you will be playing with #{player_2_symbol} as your symbol"
+require './lib/player.rb'
+require './lib/board.rb'
+require './lib/game.rb'
 
-# Display board
-puts '___1_|__2__|__3__'
-puts '___4_|__5__|__6__'
-puts '___7_|__8__|__9__'
+puts 'Welcome to Tic Tac Toe'
+def start
+  @game_symbols = %w[X O]
+  puts 'player 1 what is your name?'
+  pl_one = gets.chomp
+  puts " #{pl_one} choose your symbol either X or O to play "
+  symbol = gets.chomp
```

Comment:
You are asking the player to input X or O but you don't have a validation to check. Right now I can input anything I want or leave it empty. 

To replicate the error, type anything on the player's name **twice** the second time won't validate anything and let you keep the input.

If you type `x` or `o` in lower caps, the game doesn't recognize it as a valid input.

![Screenshot from 2020-04-27 13-24-54](https://user-images.githubusercontent.com/51926937/80406994-87986f80-888a-11ea-8063-23d9a808846b.png)

You should check all possible options and make sure players know what their symbol is.